src/Bedrock.Framework.Experimental/Protocols/Tls/Certificates/CertificateType.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Tls.Certificates
+{
+    //https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-16
+    //TLS SignatureAlgorithm Registry
+    internal enum CertificateType
+    {
+        anonymous = 0,
```

Comment:
Casing?

Reply:
Changed (it was matching the spec originally but I don't mind either way)